packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import onClickOutside from 'react-onclickoutside';
+import styles from './HookshotContent.scss';
+
+const cx = classNames.bind(styles);
+/**
+ * Key code value for the escape key.
+ */
+const KEYCODES = {
+  ESCAPE: 27,
+};
+
+const propTypes = {
+  /**
+   * The children to be displayed as content within the content.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * The function callback when am escape keydown event occurs..
+   */
+  onEsc: PropTypes.func,
+  /**
+   * The function callback when a click outside event occurs.
+   */
+  onOutsideClick: PropTypes.func,
+  /**
+   * The function callback when a resize event occurs.
+   */
+  onResize: PropTypes.func,
+  /**
+   * The function returning the frame html reference.
+   */
+  refCallback: PropTypes.func,
+};
+
+const defaultProps = {
+  onEsc: null,
```

Comment:
Should these functions default to undefined as opposed to null? I feel like having null handlers can cause issues.